ci_framework/roles/env_op_images/tasks/main.yml

```diff
@@ -0,0 +1,92 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Ensure directory is present
+  ansible.builtin.file:
+    path: "{{ cifmw_env_op_images_dir }}"
+    state: directory
+  loop:
+    - artifacts
+    - logs
+
+- name: Check if OpenStackControlPlane is setup
+  environment:
+    KUBECONFIG: "{{ cifmw_openshift_kubeconfig }}"
+    PATH: "{{ cifmw_path }}"
+  ansible.builtin.shell: |
+    oc get OpenStackControlPlane -n {{ cifmw_install_yamls_defaults['NAMESPACE'] }} --no-headers=true | awk '{print $2}'
+  register: openstack_installed
+
+- name: Get openstack operator images
+  when:
+    - cifmw_openshift_kubeconfig is defined
+    - openstack_installed.stdout | bool
```

Comment:
just wondering how that is meant to work...  does the output of this command ever give us a boolean value?

e.g in an environment with no controlplane:

```
$ oc get OpenStackControlPlane -n openstack --no-headers=true | awk '{print $2}'
No resources found in openstack namespace.
```

so that one would evaluate to false with |bool
when would it be true though? i am missing what is given from the awk '{print $2}' probably?

Reply:
This is the output when  OpenStackControlPlane CR is present in the cluster and is successfully deployed.
```
$ oc get OpenStackControlPlane -n openstack --no-headers=true | awk '{print $2}'
True
```
So, it will collect the container images only when the cluster is deployed

Reply:
iirc, if it's not an empty string, it will be considered as true.. Maybe we can get a stronger check on that command output?